utils/crypto_test.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"testing"
+)
+
+func TestEnc(t *testing.T) {
+	data := []struct {
+		input string
+		key   string
+	}{
+		{"Foo", "Boo"},
+		{"Bar", "Car"},
+		{"Bar", ""},
+		{"", "Car"},
+		{"Long input with more than 16 characters", "Car"},
+	}
+	for _, d := range data {
+
+		enc, err := EncryptString(d.input, d.key)
```

Comment:
This isn't really a unit test, as you test the encrypt and the decrypt in the same function. Instead, create a constant with the encrypted string set in stone

Reply:
Also, what about AES?